metrics/report/report_dockerfile/scaling.R

```diff
@@ -219,6 +221,12 @@ mem_stats_plot = suppressWarnings(ggtexttable(data.frame(rstats),
 	rows=NULL
 	))
 
+if (step != 1) {
+	xlabel = paste("pods x", step)
```

Comment:
I'm going to have a peek at the output, but I thought we could use the `n_pods$Result` from each entry to get the actual number of pods launched, and then use that on the X axis.
We may have to cast a bit of 'magic' on the X axis at some point, as by default I think it will split it into something like 4 or 10 divisions and number it as such, which makes for some oddities like seeing a '2.5 pods' mark now and then, which just does not make sense ;-)  Last time I looked at this it is slightly painful to get the scales to look good for all our ranges of pods (from say 5 pods to 1000 pods) - but, we can try at some point.

Reply:
I can try to play with that. boot_line_plot uses two data.frames, but I just need to make sure n_pods$Result gets propagated through